### DISCUSSION TITLE
Can the command line be filled with database roles?

### DISCUSSION THREAD

**Discussion by GG668** [dbeaver/dbeaver#14218] [Q&A]
I want to fill the oracle roles via command line, what should I do

**Comment by LonwoLonwo** [comment]
Hello @GG668 
What do you mean by "fill the roles"? Tell me more, please, about your idea.

**Reply by GG668** [reply]
yes
dbeaver.exe -con "driver=oracle|name=ori|host=127.0.0.1|port=1521|user=system|password=123456|database=XE|role=SYSOPER"
Doesn't seem to work

**Comment by LonwoLonwo** [comment]
Ok. So DBeaver just does not support this connection parameter. You can create a ticket as a feature request here: https://github.com/dbeaver/dbeaver/issues
